README.md

```diff
@@ -14,6 +14,25 @@ We hope that this engine will inspire you to experiment and create your own IAC
 
 For more information, see the [Terragrunt IAC Engine RFC](https://github.com/gruntwork-io/terragrunt/issues/3103).
 
+## Usage
+
+To utilize the OpenTofu Engine in your Terragrunt configuration, you need to specify the `engine` in HCL code.
+Here's an example:
+
+```hcl
+engine {
+  source  = "github.com/gruntwork-io/terragrunt-engine-opentofu"
+  // Specify a fixed version if you require stable behavior
```

Comment:
```suggestion
  // Specify a fixed version if you want to pin a specific engine version instead of always
  // using the latest version of the engine.
```